infra/gcp/ensure-prod-storage.sh

```diff
@@ -79,7 +85,7 @@ for prj in "${ALL_PROJECTS[@]}"; do
     color 6 "Empowering image promoter to GCR: ${prj}"
     for r in "${PROD_REGIONS[@]}"; do
         color 3 "region $r"
-        empower_artifact_promoter "${prj}" "${r}"
+        empower_artifact_promoter "${PROMOTER_SVCACCT}" "${prj}" "${PERMISSIONS}" "${r}"
```

Comment:
I'm not sure but you'll only get the first element of PERMISSIONS. 
```suggestion
        empower_artifact_promoter "${PROMOTER_SVCACCT}" "${prj}" "${PERMISSIONS[@]}" "${r}"
```
instead ?

Reply:
Yes... and also, if you use that will it get splatted and mess up the number of arguments being passed to `empower_artifact_promoter`?